README.md

```diff
@@ -122,210 +179,317 @@ sudo usermod -a -G docker user
 
 ## Installation
 
-Once all installation prerequisites have been met, WP Local Docker is installed as a global npm package by running
-`npm install -g wp-local-docker`. You can confirm it has been installed by running `10updocker --version`.
+Getting WP Local Docker up and running on your system involves a few simple steps. Start by ensuring that you have met all the [prerequisites](#prerequisites) for installation.
+
+Next, you can install WP Local Docker as a global npm package. Run the following command in your terminal:
+
+```bash
+npm install -g wp-local-docker
+```
+
+To confirm that WP Local Docker has been installed successfully, you can check its version. Simply type:
+
+```bash
+10updocker --version
+```
 
 ### Configuration
 
-The first time you run a WP Local Docker command, default configuration settings will be used if you have not manually
-configured WP Local Docker beforehand. By default, WP Local Docker will store all environments within the
-`~/wp-local-docker-sites` directory and try to manage your hosts file when creating and deleting environments. If you
-would like to customize the environment path or opt to not have WP Local Docker update your hosts file, run
-`10updocker configure` and follow the prompts.
+The first time you use a WP Local Docker command, it will use default configuration settings. These settings include storing all your environments within the `~/wp-local-docker-sites` directory and managing your hosts file when creating and deleting environments.
+
+If you want to customise the environment path or choose not to let WP Local Docker update your hosts file, you can do so. Run the command:
+
+```bash
+10updocker configure
+```
+
+Then, follow the prompts to customise your settings.
 
 ### Updating
 
-To update WP Local Docker, run `npm install -g wp-local-docker` again and NPM will install the latest version over your current one.
+Keeping WP Local Docker up-to-date is important. To update it, you can run the installation command again. NPM will replace your current version with the latest one. Here's the command:
```

Comment:
This section should link to "Upgrade an environment"

Reply:
I don't think there's a benefit to linking to the upgrade section here. Upgrading environments and updating WP Local Docker are two separate actions that needn't happen together. 

If they're required together for a major update, as is the case with `4.0.0`, then we should call that out in the updating guide.